src/socket.cpp

```diff
@@ -188,12 +202,24 @@ void CSocket::SendPacket ( const CVector<uint8_t>& vecbySendBuf, const CHostAddr
         UdpSocketOutAddr.sin_port        = htons ( HostAddr.iPort );
         UdpSocketOutAddr.sin_addr.s_addr = htonl ( HostAddr.InetAddr.toIPv4Address() );
 
-        sendto ( UdpSocket,
-                 (const char*) &( (CVector<uint8_t>) vecbySendBuf )[0],
-                 iVecSizeOut,
-                 0,
-                 (sockaddr*) &UdpSocketOutAddr,
-                 sizeof ( sockaddr_in ) );
+        if (
+            sendto ( UdpSocket,
+                     (const char*) &( (CVector<uint8_t>) vecbySendBuf )[0],
+                     iVecSizeOut,
+                     0,
+                     (sockaddr*) &UdpSocketOutAddr,
+                     sizeof ( sockaddr_in ) )
+            < 0 )
```

Comment:
style

Reply:
How do I run the Coding Style check locally?

Reply:
You can do it with clang-format. I‘m not sure about the exact command, but maybe it’s something with clang-format -e ? However, you‘ll probably need to update this PR to the current master (= rebase your git branch).

Reply:
Ok I’ll check clang-format, and also what rebase means lol

Reply:
Maybe this helps: https://learnxinyminutes.com/docs/git/